src/tools/rpgmodel.h

```diff
@@ -0,0 +1,74 @@
+#ifndef RPGMODEL_H
+#define RPGMODEL_H
+
+#include <QObject>
+#include <QAbstractItemModel>
+#include <data.h>
+
+template <class DATA>
```

Comment:
Yep, that's it ;)
I love template programming, but why I use it here, is because we can use it easily in the editor now (when you open qt-editor and open the `variableoperations.ui` in line 49, you see `VariableRpgComboBox`. That is so neat, to use it like that, instead of using a generic qcombobox and insert all the data in the cpp file. You also could achieve this with subclassing, but hmm... don't like subclassing that much... one could argue about that